calcom/cal.com
# Design issues on success page in some languages

## What does this PR do?

On the Success Page, the divs were not aligned in all languages.

Fixes #3489 

## Type of change



## How should this be tested?

Just set up a meeting, and then on the Success Page, confirm that everything is aligned.

### Loom Videos:

Before: https://www.loom.com/share/b4c4cde24bfe45779863dcbd141eba25
Later, with the margin in the mobile version: https://www.loom.com/share/fd7a0e0120f1421f96f0dc2eb52a856d
